addons/web/static/tests/services/data_manager_tests.js

```diff
@@ -0,0 +1,130 @@
+odoo.define('web.data_manager_tests', function (require) {
+    "use strict";
+
+    const DataManager = require('web.DataManager');
+    const rpc = require('web.rpc');
+    const config = require('web.config');
+    const testUtils = require('web.test_utils');
+
+    /**
+     * Create a simple data manager with mocked functions:
+     * - mockRPC -> rpc.query
+     * - isDebug -> config.isDebug
+     * @param {Object} params
+     * @param {Function} params.isDebug
+     * @param {Function} params.mockRPC
+     * @returns {DataManager}
+     */
+    function createDataManager({ isDebug, mockRPC }) {
+        testUtils.mock.patch(rpc, { query: mockRPC });
+        testUtils.mock.patch(config, { isDebug });
+
+        const dataManager = new DataManager();
+        return dataManager;
+    }
+
+    /**
+     * Helper function returning an object similar to the return value of a server
+     * 'load_views'.
+     * @param {Object} params
+     * @param {Object} [params.args]
+     * @param {Object} params.kwargs
+     * @param {Object} [params.model]
+     * @returns {Object}
+     */
+    function mockLoadViews({ args, kwargs, model }) {
```

Comment:
you can use the MockServer for this, I think

Reply:
It looks like MockServer does not have a method that properly mocks the return value of a 'load_views' sent to the server, nor of any "filters" related method (`get_filters`, `create_or_replace`, etc.). I have done some tinkering in this helper but I'm still unsure about it... Any idea on how to make this cleaner/more usable ?

Reply:
Damn you're right, the MockServer mocks the DataManager as well (of course...). I guess it's fine then.